graphql-schema/reaction.graphql

```diff
@@ -0,0 +1,23 @@
+# TODO: uncomment this when other types are ready
+# union ReactionTarget = Proposal
+
+type Reaction implements Node {
+  id: ID!
+  target: Proposal!
+  reaction: String!
+  address: String!
+}
+
+input CreateProposalReactionInput {
+  proposalId: ID!
+  reaction: String!
+}
+
+type ReactionCount {
+  count: Int!
+  reaction: String!
+}
+
+extend type Mutation {
+  createProposalReaction(input: CreateProposalReactionInput!): Reaction! @authed
```

Comment:
`setReaction`?

& we can try to make this generic by acting on node id? so the resolver should parse the item type & entity id